packages/web-components/src/components/va-radio/va-radio.tsx

```diff
@@ -250,7 +260,7 @@ export class VaRadio {
           aria-label={ariaLabel}
         >
           <legend part="legend">
-            {label}
+            {Header ? <Header part="header">{label}</Header> : label}
```

Comment:
That's a good point and valid. The header level will depend on how it's implemented on the page which could require a change in font-size to match the designs.

Reply:
This is a good thing to prepare for just in case. I can see some of our legends being extra long; our `h1`s are pretty chonky too-- especially on mobile 😆